src/System.Net.WebHeaderCollection/src/System/Net/WebHeaderCollection.cs

```diff
@@ -131,7 +132,9 @@ private bool AllowHttpResponseHeader
             }
         }
 
-        public override void Set(string name, string value)
+#pragma warning disable CS8610 // Supress warning: Nullability of parameter 'name' doesn't match overridden member
```

Comment:
Nit: it's obvious you're suppressing a warning... you don't need the "Supress warning: " part of the comment :smile: